config/locales/help_text/fr.yml

```diff
@@ -12,18 +12,17 @@ fr:
         informations avec <strong>%{sp}</strong> pour connecter votre compte.
       ial1_intro_html: Nous partagerons vos informations avec <strong>%{sp}</strong>
         pour connecter votre compte.
-      ial2_consent_reminder_html: '<strong>%{sp}</strong> doit savoir qui vous êtes
-        pour se connecter à votre compte. Vous devez consentir chaque année à
-        partager vos informations vérifiées avec <strong>%{sp}</strong>. Nous
-        partagerons ces informations :'
+      ial2_consent_reminder_html: '<strong>%{sp}</strong> a besoin de savoir qui vous
+        êtes pour se connecter à votre compte. Vous devez consentir chaque année
+        à partager vos informations vérifiées avec <strong>%{sp}</strong>. Nous
+        partagerons ces informations : '
```

Comment:
Is the trailing space expected? (2 more of the same in this file)

Reply:
Fixed in 5f30149